turbopy/core.py

```diff
@@ -262,6 +262,8 @@ def read_diagnostics_from_input(self):
                     # Values in di supersede values in params because
                     # of the order in which these are combined
                     di = {**params, **di, "type": diag_type}
+                    if "output_type" not in di:
+                        di = {**di, "output_type": "out"}
```

Comment:
The variable `input_data["output_type"]` is not really the same thing as a file ending. I think it's ok to use "out" as a default (in the line `file_end = di.get("output_type", "out")` below), but I don't really think it's "correct" to save "out" as the `"output_type"` of the diagnostic.

Reply:
Oh okay.